### ISSUE TITLE
Whether dtls1.3 is supported

### ISSUE THREAD

**Issue by redoriental** [wolfSSL/wolfssl#6236]

**Comment by kareem-wolfssl** [comment]
Yes, we support DTLS 1.3.  Pass `--enable-dtls13` to ./configure to enable this.
Check out our DTLS 1.3 examples here: https://github.com/wolfSSL/wolfssl-examples/tree/master/dtls
